src/cahillKeyes.js

```diff
@@ -0,0 +1,713 @@
+/*
+ * Cahill-Keyes projection
+ *
+ * Implemented in Perl by Mary Jo Graça (2011)
+ *
+ * Ported to D3.js by Enrico Spinielli (2013)
+ *
+ */
+import { abs, asin, atan2, cos, degrees, max, min, pi, radians, sin, sqrt, tan } from "./math";
+import polyhedral from "./polyhedral/index";
+import { geoProjectionMutator as projectionMutator } from "d3-geo";
```

Comment:
Re-use code from `d3-geo`
`import { cartesian, spherical } from "d3-geo";`
and remove the L142-L155

Reply:
👍